CMSgov/eAPD
# Tbolt/4182 Fix issue with renewing state admin

Resolves #4182 

### Description
Updates the state admin matching process to allow matching on already existing matches.

### Significant changes or possible side effects
State admins who have been previously approved should now be eligible "matches." Unsure if any side effects are introduced from this.

### Automated test cases written
Test added to verify that state admin role will return a match

### Steps to manually verify this change
1. Login as fed admin
2. For a already valid state admin user for this current FFY, add a new certification for next FFY
3. Verify that the certification has a match
4. Verify that you can make the match successfully


### This pull request is ready to review when


### This pull request can be merged when

- [x] QA has verified the accesibility using tools such as screen readers and WAVE (accessibility evaluation tool)
